README.md

```diff
@@ -1,88 +1,78 @@
 # PromoTube
 
 PromoTube is a webapp that allows users to easily find promo codes and affiliate links that go with their favorite creators or products.
```

Comment:
PromoTube is a browser-based web application which extracts promotional codes and affiliate links from YouTube video descriptions.